plutus-core/src/Language/PlutusCore/Evaluation/Machine/Ck.hs

```diff
@@ -65,8 +66,8 @@ data CkValue uni fun =
    case anyway (see Note [Ignoring context in UserEvaluationError] in
    Exception.hs)
 -}
-mkBuiltinApplication :: () -> fun -> Arity -> [Type TyName uni ()] -> [Term TyName Name uni fun ()] -> Term TyName Name uni fun ()
-mkBuiltinApplication () bn arity0 tys0 args0 =
+mkBuiltinApplication :: fun -> Arity -> [Type TyName uni ()] -> [Term TyName Name uni fun ()] -> Term TyName Name uni fun ()
+mkBuiltinApplication bn arity0 tys0 args0 =
```

Comment:
Removed all `()` arguments from functions. I can bring them back if anybody cares. I don't care either way, so went for "less code to think about".